src/main/java/frc/robot/commands/SnapToYawCommand.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+import frc.robot.DriveInstruction;
+import frc.robot.RobotConstants;
+import frc.robot.subsystems.DriveSubsystem;
+import frc.robot.subsystems.NavXSubsystem;
+
+/**
+ *
+ * @author aryan
+ */
+public class SnapToYawCommand extends EntechCommandBase{
```

Comment:
this command will have a timeout of 33 seconds ( the default). I think that's way too long for this.  If something goes very, very wrong with this ( possible since we've got something complex like a PID controller in here), we should timeout more quickly i think.
this would apply to all commands that are in auto ( max 15 seconds, no sense having a command that can run longer than that),